consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/JobStatus.java

```diff
@@ -0,0 +1,74 @@
+package com.redhat.lightblue.migrator.consistency;
+
+/**
+ *
+ * JobStatus can be one of the following values:
+ *      NEW                - Means the Job just created and it isn't ready to be executed
+ *      READY              - Means the Job is ready to be executed but not in execution
+ *      STARTING           - It just got to be processing
+ *      RUNNING            - Currently being processed for synchronous request
+ *      RUNNING_ASYNC      - Currently being processed for asynchronous request
+ *      FINISHING          - The execution is almost complete, just few more steps to move to a final status
```

Comment:
Having status of FINISHING and ABORTING requires additional writes to the entity.  I don't know that there's sufficient value to track these.  If a job is either finishing or aborting I assume it will very quickly be done.  There shouldn't be any cleanup of data to do, so no external system interactions to slow things down.  It's all in memory, and not sure exactly what would be *done* after these states.